src/services/dashboards/widgets/_base/base-pie/BasePieWidget.vue

```diff
@@ -68,18 +72,27 @@ const state = reactive({
         return GROUP_BY_ITEM_MAP[groupBy]?.label ?? groupBy;
     }),
     chartType: computed(() => state.options.chart_type ?? CHART_TYPE.PIE),
-    chartData: computed<Data[]>(() => {
+    chartItems: computed<Item[]>(() => {
```

Comment:
nitpick: Actually, the `chartData` variable name is convention for widgets :)
So I suggest not to change this name..
I think that you changed the name because `chartData` is used for `items` prop which is bound to WidgetDataTable.
But the reason why `chartData` is bound to `items` prop in this widget is that table items are the same with `chartData`.
But in other cases, `tableItems` would be needed.